worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/FastSchematicReaderV3.java

```diff
@@ -87,6 +89,7 @@ public class FastSchematicReaderV3 implements ClipboardReader {
 
     private VersionedDataFixer dataFixer;
     private BlockVector3 offset;
+    private BlockVector3 origin;
```

Comment:
should default to `BlockVector3.DEFAULT` (see the other comment for reason)

Reply:
Implemented default value